jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt

```diff
@@ -41,7 +45,13 @@ class OpenAwsLocalTerminal : DumbAwareAction(message("aws.terminal.action"), mes
                         return@executeOnPooledThread
                     }
                     runInEdt {
-                        val runner = AwsLocalTerminalRunner(project, connection.region, credentials)
+                        val runner = AwsLocalTerminalRunner(project) { envs ->
```

Comment:
Do we not have tests for this action?

Reply:
:( no - we probably should.

*sigh* will add.

Reply:
(well actually that's not true - we have a UI test)